wiki/Tournaments/CH/CHS_1/en.md

```diff
@@ -0,0 +1,524 @@
+
+- NoMod
+  - [PassCode - Ray (SkyFlame) \[Resonance\]](https://osu.ppy.sh/beatmapsets/1180927#osu/2462350)
+  - [IOSYS feat. Nanahira - Judgment Day Has Come (-Tynamo) \[BMD's Expert\]](https://osu.ppy.sh/beatmapsets/1152650#osu/2405862)
+  - [BUTAOTOME - Warp On (BarkingMadDog) \[Extreme\]](https://osu.ppy.sh/beatmapsets/889029#osu/1858228)
+  - [Endorfin. - Luminous Rage -Feryquitous OrderBless Remix- (Nokris) \[Obscuring Tranquility\]](https://osu.ppy.sh/beatmapsets/743239#osu/1567435)
+  - [MAX MAXIMIZER vs. DJ TOTTO - Rebellio (Realazy) \[TOYBOT'S EXPERT\]](https://osu.ppy.sh/beatmapsets/744636#osu/1623397)
+  - [Virt - Choppastyle (YokesPai) \[KIRBY Mix\]](https://osu.ppy.sh/beatmapsets/1023284#osu/2140702)
+- Hidden
+  - [Morimori Atsushi - Tits or get the fuck out!! (EvilElvis) \[Chastity\]](https://osu.ppy.sh/beatmapsets/352169#osu/776107)
+  - [Yamai - E? Aa, Sou. (Faito) \[?\]](https://osu.ppy.sh/beatmapsets/1245259#osu/2588430)
+  - [Zekk - SUMMER (Typ4) \[Paradise\]](https://osu.ppy.sh/beatmapsets/971592#osu/2034284)
+- HardRock
+  - [XX:me - Escape (jonathanlfj) \[Freedom\]](https://osu.ppy.sh/beatmapsets/811119#osu/1701563)
+  - [DragonForce - Ashes of the Dawn (pw384) \[Extra\]](https://osu.ppy.sh/beatmapsets/786141#osu/1650328)
+  - [tarolabo - Zeitmesser (Charles445) \[Expert\]](https://osu.ppy.sh/beatmapsets/310339#osu/693542)
+- DoubleTime
+  - [Utagumi Setsugekka - Yumemi Sunrise (Zweib) \[Insane\]](https://osu.ppy.sh/beatmapsets/132312#osu/338862)
+  - [Grynpyret - Boba Beach (VINXIS) \[Habanero\]](https://osu.ppy.sh/beatmapsets/327224#osu/726289)
+  - [Kola Kid - timer (Mirash) \[Insane\]](https://osu.ppy.sh/beatmapsets/1089084#osu/2277128)
+  - [zts - miragecoordinator (Tsumugi-chan) \[Collab Expert\]](https://osu.ppy.sh/beatmapsets/1085924#osu/2270798)
+- FreeMod
+  - [umu. - humanly (Luscent) \[Expert\]](https://osu.ppy.sh/beatmapsets/1045518#osu/2185651)
+  - [SWAN K feat. Asuka M - LOVE B.B.B (\[ Couch \] Lite) \[EXTRA\]](https://osu.ppy.sh/beatmapsets/1100957#osu/2433110)
+  - [Feryquitous feat. Aitsuki Nakuru - Kairikou (Ryuusei Aika) \[Skystar's Irregular\]](https://osu.ppy.sh/beatmapsets/740862#osu/1723257)
+- Tiebreaker
+  - **[Unlucky Morpheus - Angreifer (Scarsnic) \[Zeitlos\]](https://osu.ppy.sh/beatmapsets/581468#osu/1230741)**
+  - **[Camellia - LET'S JUMP (Camellia's "diffractive" Hardcore Remix) (Moko) \[Isomorphic Substitution\]](https://osu.ppy.sh/beatmapsets/1052797#osu/2326935)**
+  - **[GET IN THE RING - Midnight Syndrome (Atalanta) \[Desire\]](https://osu.ppy.sh/beatmapsets/862929#osu/1886627)**
+
+### Finals
+
+**[Download the mappack here! (156 MB)](https://drive.google.com/file/d/1g847SBMY7dh1rO1PD4cGzIWu_8ymbunN/view)**
+
+- NoMod
+  - [Twin-Tails - Twintail Dreamer! (Minorsonek) \[diuS' Expert\]](https://osu.ppy.sh/beatmapsets/773026#osu/1680262)
+  - [GLORYHAMMER - Rise of the Chaos Wizards](https://osu.ppy.sh/beatmapsets/990188#osu/2164735)
```

Comment:
diff is missing

Reply:
Which the mapper probably deleted the map after the map's been used, what's the best action here?

Reply:
if it can't be accessed then its fine as it is i suppose